docs/migrations/v6-to-v7.md

```diff
@@ -290,4 +290,73 @@ import (
 
 - host.RouterKey
 + ibcexported.RouterKey
-```
\ No newline at end of file
+```
+
+## Upgrading to Cosmos SDK 0.47
+
+The following should be considered as complementary to [Cosmos SDK v0.47 UPGRADING.md](https://github.com/cosmos/cosmos-sdk/blob/v0.47.0-rc2/UPGRADING.md).
+
+### Protobuf 
+
+Protobuf code generation, linting and formatting have been updated to leverage the `ghcr.io/cosmos/proto-builder:0.11.5` docker container. IBC protobuf definitions are now packaged and published to [buf.build/cosmos/ibc](https://buf.build/cosmos/ibc) via CI workflows. The `third_party/proto` directory has been removed in favour of dependency management using [buf.build](https://docs.buf.build/introduction).
+
+### App modules
+
+The return values of `RandomizedParams` and `ProposalContents` have been updated.
+
+```diff
+// ProposalContents doesn't return any content functions for governance proposals.
+- func (AppModule) ProposalContents(_ module.SimulationState) []simtypes.WeightedProposalContent {
++ func (AppModule) ProposalContents(_ module.SimulationState) []simtypes.WeightedProposalMsg {
```

Comment:
Actually, the return value from `ProposalContents` have not been updated. We've created an additional method that can define a proposal msg for simulation:
- https://github.com/cosmos/cosmos-sdk/blob/a9c18cd47f923d1a4b6ceac03b1dcef7cec03a5e/types/module/simulation.go#L29-L39